images/linux/scripts/helpers/install.sh

```diff
@@ -50,6 +52,20 @@ verlte() {
     [  "$1" = "$sortedVersion" ]
 }
 
+verify_http_status_code()
+{
+    URL="$1"
+    http_code=$(curl -s -o out.html -w '%{http_code}' $URL;)
+
+    if [[ $http_code -ge 400 ]]; then
+        cat out.html
```

Comment:
I don't think it's necessary to have this output, it can be useless. E.g: https://bing.com/404